### ISSUE TITLE
How to change the words and language.

### ISSUE THREAD

**Issue by Sparkkkkie** [Kovah/Taboo#37]
Hello there! I would like to make my own wordlist for this, is there any way i can do that? Thanks.

**Comment by Kovah** [comment]
Currently not, as all words are managed by the [taboo-data](https://www.npmjs.com/package/taboo-data) library.

**Comment by Sparkkkkie** [comment]
> Currently not, as all words are managed by the [taboo-data](https://www.npmjs.com/package/taboo-data) library.

I also cannot edit that in some way?

**Comment by Kovah** [comment]
Not really. I mean it would be possible by building the game on your own after making significant changes, but I cannot guarantee anything will work after that.
